src/components/member/tabs/payments-tab/index.tsx

```diff
@@ -188,32 +185,24 @@ const PaymentsTabComponent: React.FC<WithShowNotification & {
         <GenerateSetupDirectDebitLink memberId={memberId} />
       </Spacing>
 
-      {data.member?.directDebitStatus?.activated && (
-        <Mutation mutation={CHARGE_MEMBER_MUTATION}>
-          {(chargeMember) => (
-            <>
-              <ThirdLevelHeadline>Charge:</ThirdLevelHeadline>
-              <Spacing bottom="small">
-                <Input
-                  type="number"
-                  value={amount}
-                  onChange={(e) => setAmount(e.target.value)}
-                  label={data.member?.contractMarketInfo?.preferredCurrency}
-                  labelPosition="right"
-                  placeholder="ex. 100"
-                />
-              </Spacing>
-              <Button
-                disabled={!amount}
-                variation="primary"
-                onClick={() => handleChargeSubmit(chargeMember)}
-              >
-                Submit
-              </Button>
-            </>
-          )}
-        </Mutation>
-      )}
+      {data.member?.directDebitStatus?.activated &&
+        Number(account?.currentBalance.amount) > 0.0 && (
+          <Mutation mutation={CHARGE_MEMBER_MUTATION}>
+            {(chargeMember) => (
+              <>
+                <ThirdLevelHeadline>
+                  Charge member's current balance:
+                </ThirdLevelHeadline>
+                <Button
+                  variation="primary"
+                  onClick={() => handleChargeSubmit(chargeMember)}
+                >
+                  Charge {formatMoney(account?.currentBalance!)}
+                </Button>
+              </>
+            )}
+          </Mutation>
+        )}
```

Comment:
I suggest we display the balance if it's not positive, instead of not showing anything
```suggestion
      {data.member?.directDebitStatus?.activated  && (
      <>
        <ThirdLevelHeadline>
          Charge member's current balance:
        </ThirdLevelHeadline>
         {Number(account?.currentBalance.amount) > 0.0 ? (<Mutation mutation={CHARGE_MEMBER_MUTATION}>
            {(chargeMember) => (
              <>
                <Button
                  variation="primary"
                  onClick={() => handleChargeSubmit(chargeMember)}
                >
                  Charge {formatMoney(account?.currentBalance!)}
                </Button>
              </>
            )}
          </Mutation>) : 'Unable to charge member since the balance is ${formatMoney(currentBalance)}'}
      </>
        )}
```

Reply:
Good point, have applied you stuff.
Also added that we wait for the account to be loaded, to not get errors while it's "undefined"